app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,37 @@
+class QueueItemsController < ApplicationController
+  before_filter :require_user
+ 
+  def index
+    @queue_items = current_user.queue_items#.order(order: :asc)
+  end
+
+  def create
+    video = Video.find(params[:video_id])
+    queue_video(video)
+    redirect_to my_queue_path
+  end
+
+
+  def destroy
+    queue_item = QueueItem.find(params[:id])
+    queue_item.destroy! if current_user.queue_items.include?(queue_item)
+    flash[:notice] = "destroy the queue item"
```

Comment:
You might want to change the `flash` to work with bootstrap 3. Perhaps `flash[:info]`. See [here](http://getbootstrap.com/components/#alerts)